ODF-community.french.md

```diff
@@ -0,0 +1,526 @@
+sur le projet.</td>
+</tr>
+</table>
+
+<br/>
+
+Les décisions ouvertes sont issues des principes open source <a name="open-decisions-open-principles"></a>
+-----------------------------------------------------
+
+_page 4 sur 15_ 
+
+**Echanges ouverts**
+
+Que vous developpiez un logiciel ou que vous essayiez de résoudre un problème
+dans l'entreprise, l'échange ouvert commence en partageant le "code source" avec
+d'autres personnes. Un échange libre des idées est essentiel pour créer un
+environement où chacun a les moyens d'apprendre et d'utiliser l'information
+existante pour imaginer de nouvelles idées. 
+
+**Participation**
+
+Quand nous sommes libres de collaborer, nous créons. Nous pouvons résoudre des
+problèmes qu'un individu ne pourrait pas résoudre tout seul. Et quand nous
+pouvons implémenter des standards ouverts, nous permettons aux autres de
+participer dans le futur. 
+
+**Publier tôt + souvent**
+
+Le prototypage rapide peut mener à des échecs rapides, et cela conduit à de
+meilleures solutions. Quand on est libre d'expérimenter, on peut regarder les
+problèmes de manières différentes et chercher des réponses dans des endroits
+différents. On peut apprendre en faisant.
+
+**Méritocracie**
+
+Dans un méritocracie, les bonnes idées peuvent venir de n'importe où et la
+meilleure idée gagne. Tout le monde a accès à la même information. C'est la
+qualité du travail qui détermine quels projets prennent de l'ampleur et attire
+le support et les efforts de la communauté.
+
+**Communauté**
+
+Les communautés sont formées autour d'un objectif commun. Elles rassemblent des
+idées diverses et partagent le travail. Une fois réunie, une communauté globale
+peut créer au delà des capacités de n'importe quel individu. Elle multiplie les
+efforts et réparti le travail. Ensemble nous pouvons faire plus.
+
+Adapté de : https://opensource.com/open-source-way
+
+<br/>
+
+Comment les principes open source conduisent à de meilleures décisions <a name="how-open-source-principles"></a> 
+---------------------------------------------------
+
+_page 5 sur 15_ 
+
+<table>
+<tr>
+<td>Principes</td>
+<td>→</td>
+<td>Pratique</td>
+<td>→</td>
+<td>Résultats</td>
+</tr>
+<tr>
+<td>
+• Échanges ouverts<br/> • Participation<br/> • Publier tôt + souvent<br/> •
+Méritocracie<br/> • Communauté
+</td>
+<td>→<br/>→<br/>→</td>
+<td>
+• Transparence avec les usagers internes et les autres parties prenantes <br/> •
+Implication des usagers<br/> • Récolter du feedback et s'adapter de manière
+itérative<br/> • Imaginer avec les usagers<br/> • Construire de la confiance et du respect via la collaboration 
+</td>
+<td>→<br/>→<br/>→</td>
+<td>
+• Adhésion des usagers<br/> • Adoption plus rapide et plus forte<br/> • Les
+meilleures idées gagnent<br/> • Moins de bugs, de problèmes et de impacts
+imprévus <br/> • Plus grand engagement des collaborateurs <br/> • Décisions
+alignées sur le stratégie et la culture
```

Comment:
Original sentence is ``Decisions aligned to strategy and culture`` so I guess it is implied that we talking about the strategy from Red Hat